docs/sql-ref-syntax-aux-show-table.md

```diff
@@ -18,5 +18,147 @@ license: |
   See the License for the specific language governing permissions and
   limitations under the License.
 ---
+### Description
 
-**This page is under construction**
+SHOW TABLE EXTENDED will show information for all tables matching the given regular expression.
+Output includes basic table information and file system information like `Last Access`, 
+`Created By`, `Type`, `Provider`, `Table Properties`, `Location`, `Serde Library`, `InputFormat`, 
+`OutputFormat`, `Storage Properties`, `Partition Provider`, `Partition Columns` and `Schema`.
+
+Users cannot use regular expression for table name if a partition specification is present and  it will output 
+the given partition's file specific system information such as `Partition Parameters` and `Partition Statistics` 
+
+### Syntax
+{% highlight sql %}
+SHOW TABLE EXTENDED [IN|FROM database_name] LIKE 'identifier_with_wildcards' [PARTITION(partition_spec)];
+{% endhighlight %}
+
+### Parameters
+<dl>
+ <dt><code><em>IN|FROM database_name</em></code></dt>
+  <dd>
+    Specifies database name. If not provided, will use the current database.
+  </dd>
+  <dt><code><em>LIKE string_pattern</em></code></dt>
+  <dd>
+    Specifies a string pattern that is used to match the table in the database. 
```

Comment:
Is it a full regex? we should say a little more about what's allowed here.

Reply:
updated the details